lib/chef/resource/support/client.erb

```diff
@@ -20,15 +20,15 @@
       @policy_group
       @policy_name
       @ssl_verify_mode).each do |prop| -%>
-<% next if instance_variable_get(prop).nil? || instance_variable_get(prop).empty? -%>
-<%=prop.delete_prefix("@") %> = <%= instance_variable_get(prop).inspect %>
+<% next if instance_variable_get(prop).nil? || instance_variable_get(prop).empty? || prop == '@log_location' -%>
```

Comment:
Since `log_location` gets special treatment below, it should be removed from the list above instead of being skipped like this.

Reply:
agreed